File: Questionnaire.Bll/Dtos/UserQuestionnaireAnswerDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Questionnaire.Bll.Dtos
{
    public class UserQuestionnaireAnswerDto
    {
        //Questionnaire Id
        public int Id { get; set; }

        public int QuestionId { get; set; }

        public int AnswerId { get; set; }

        public string UserAnswer { get; set; }
    }
}

File: Questionnaire.Api/ClientApp/src/app/features/questionnaire/models/userQuestionnaireAnswerDto.ts
export class UserQuestionnaireAnswerDto {
    id: number;
    questionId: number;
    answerId: number;
    userAnswer: string;

    constructor(id: number, questionId: number, answerId: number, userAnswer: string) {
        this.id = id;
        this.questionId = questionId;
        this.answerId = answerId;
        this.userAnswer = userAnswer;
    }
}
